### PR TITLE
add custom build settings for releaser

### PR DESCRIPTION
**Pull request by J-Thompson12**

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        - -X github.com/weaveworks/weave-gitops/cmd/wego/version.BuildTime={{.Date}}
+        - -X github.com/weaveworks/weave-gitops/cmd/wego/version.Branch=$(BRANCH) 
+        - -X github.com/weaveworks/weave-gitops/cmd/wego/version.GitCommit={{.Commit}}
+        - -X github.com/weaveworks/weave-gitops/pkg/version.FluxVersion=$(FLUX_VERSION)
+      env:
+        - CGO_ENABLED=0
+      hooks:
+        pre: export BRANCH=git rev-parse --abbrev-ref HEAD
+        pre: export CURRENT_DIR=pwd
+        pre: export FLUX_VERSION=$(CURRENT_DIR)/tools/bin/stoml $(CURRENT_DIR)/tools/dependencies.toml flux.version
```
Comment (jrryjcksn) [inline .goreleaser.yml:16]: Will `stoml` be available here? We download it in the Makefile

**Review by jrryjcksn — APPROVED** [review]
Curious about stoml :thinking:

**Review by josecordaz — APPROVED** [review]
LGTM

File: .goreleaser.yml (hunk 1 above)
Comment (J-Thompson12) [reply .goreleaser.yml:16]: You might be right. I may need to do some stuff

**Pull request merged**